app/retail/templates/shared/footer.html

```diff
@@ -17,7 +17,7 @@
 {% endcomment %}
 {% load i18n static %}
 <footer class="container-fluid no-gutters footer">
-  <div class="container footer__main">
+  <div class="container footer__main" data-src="{% static "v2/images/footer.png" %}">
```

Comment:
how this work?

Reply:
jquery-unveil.js loads it iff the user scrolls down to the footer (not upon pageload)